docs/docusaurus/docs/core/connect_to_data/configure_credentials/_access_key_pair.md

```diff
@@ -0,0 +1,24 @@
+
+To use key-pair authentication for Snowflake, you will pass the private key as a connection argument with `kwargs` in addition to passing connection details with the `connection_string` parameter. Here's an example of how to access your private key in Python.
+
+```python title="Python"
+import pathlib
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+
+PRIVATE_KEY_FILE = pathlib.Path("rsa_key.p8").resolve(strict=True)
```

Comment:
It's probably worth adding a comment here to note that the user will need to update the path to be specific to where they store their RSA key (possibly also referencing the secrets managers? I'm not sure about the best way to handle this).

Reply:
rather than add a comment, I'm going to change the placeholder value to be more obviously a placeholder (similar to other placeholder values on this page)

The next step after this one already mentions secret managers, so I'd like to see if we get customer questions about using secret managers in conjunction with key-pair auth before elaborating on it.